### PR TITLE
[stable/apm-server] Move probes to values 

### PR DESCRIPTION
**Pull request by david92rl**
#### Is this a new chart
No

#### What this PR does / why we need it:
Moves liveness and readiness probes to values file.
The apm-server integrates it's own `/healthcheck` endpoint. It might be more useful than the root `/` endpoint.
This gives enough flexibility to configure it while being backwards compatible.

#### Which issue this PR fixes
None

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/apm-server/templates/daemonset.yaml [hunk 1]
```diff
@@ -52,15 +52,9 @@ spec:
           - name: http
             containerPort: 8200
         livenessProbe:
-          httpGet:
-            path: /
-            port: http
-          initialDelaySeconds: 60
+{{ toYaml .Values.livenessProbe | indent 10 }}
```
Comment (mumoshu) [inline stable/apm-server/templates/daemonset.yaml:55]: The use of `nindent` here would make it even more readable:

```
        livenessProbe:
          {{ toYaml .Values.livenessProbe | nindent 10 }}
```

Comment (david92rl) [reply stable/apm-server/templates/daemonset.yaml:55]: Agreed, but then there will be inconsistencies with the other definitions as they are using `indent`. I could change all of them too, but I think that should go in a different PR

Comment (mumoshu) [reply stable/apm-server/templates/daemonset.yaml:55]: If you have some time, please just change things you've introduced in this PR to use `nindent` instead.

Changes for other parts should definitely go into a different PR!

**Pull request closed without merging**